tools/pd-ctl/pdctl/command/operator.go

```diff
@@ -427,6 +428,29 @@ func removeOperatorCommandFunc(cmd *cobra.Command, args []string) {
 	cmd.Println("Success!")
 }
 
+// NewHistoryOperatorCommand returns a command to history finished operators.
+func NewHistoryOperatorCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "history <start>",
+		Short: "list all finished operators since start",
+		Run:   historyOperatorCommandFunc,
+	}
+	return c
+}
+
+func historyOperatorCommandFunc(cmd *cobra.Command, args []string) {
```

Comment:
How about adding a test for `pd-ctl`? And it's better to give an example about the format of time in the help message.

Reply:
it is exist in [tests/pdctl/operator/operator_test.go](https://github.com/tikv/pd/pull/4631/files#diff-a0f6a1a99bc2f5052816b367d98307b7ed528eb7a0582c73f86e25cd35861933). The example is useful.

Reply:
I mean in the command output, you can add an example in the `Short` or `Example` field.